WordPress/src/main/res/values/key_strings.xml

```diff
@@ -356,16 +356,16 @@
     </string-array>
 
     <string-array name="site_settings_image_quality_entries" translatable="false">
-        <item>@string/media_encoder_quality_80</item>
-        <item>@string/media_encoder_quality_85</item>
-        <item>@string/media_encoder_quality_90</item>
-        <item>@string/media_encoder_quality_95</item>
-        <item>@string/media_encoder_quality_100</item>
+        <item>@string/media_encoder_quality_low</item>
+        <item>@string/media_encoder_quality_medium</item>
+        <item>@string/media_encoder_quality_high</item>
+        <item>@string/media_encoder_quality_very_high</item>
+        <item>@string/media_encoder_quality_maximum</item>
     </string-array>
 
     <string-array name="site_settings_image_quality_values" translatable="false">
+        <item>70</item>
         <item>80</item>
-        <item>85</item>
```

Comment:
I think on Android we use these values to persist the value of the _Image Quality_ setting. Removing the value `85` leads to an inconsistency in this setting if it was previously set and the app is updated with these changes.

**Steps:**
- Install a build previous to these changes ([jetpack-prototype-build-pr19581-eb71010.apk](https://d2twmm2nzpx3bg.cloudfront.net/8204deb9481772038e97669f4e11f5a5e6b4f94f/jetpack-prototype-build-pr19581-eb71010.apk)).
- Open the app and go to Me -> App Settings -> Image Quality.
- Choose _Medium_ option.
- Install the build with these changes ([jetpack-prototype-build-pr19644-452ed67.apk](https://d2twmm2nzpx3bg.cloudfront.net/8204deb9481772038e97669f4e11f5a5e6b4f94f/jetpack-prototype-build-pr19644-452ed67.apk)).
- Open the app and go to Me -> App Settings -> Image Quality.
- Observe that the value set in _Image Quality_ is _Low_. 
- Tap on _Image Quality_ and observe that no quality option is checked.

As far as I tested, the process of uploading an image is unaffected. However, I noticed that the quality used is the one originally set (85%) instead of the new one (80%). Regarding the _Low_ quality label shown in App Settings, I presume it's showing it because it defaults to the first item of the list (i.e. Low) when it can't fetch the proper value.

We might consider automatically updating the setting for users affected by this case to the new value for Medium Quality. WDYT?

Reply:
Thank you for spotting this issue, @fluiddot! I made a change that addresses it in https://github.com/wordpress-mobile/WordPress-Android/pull/19644/commits/7109245deee9ac10a906173971dfa876cc031510. As part of that change, there's now a check that ensures the quality value exists in the array. If it doesn't exist, the default value is set.

I'm thinking this approach will help us if we want to make further changes in the future.

Let me know what you think or if you perhaps see a different approach to take here.

Reply:
Confirmed this is solved in recent changes.